gateway/modules/crud/mgo/read.go

```diff
@@ -141,3 +184,93 @@ func generateSortOptions(array []string) bson.D {
 
 	return sort
 }
+
+func getGroupByStageFunctionsMap(functionsMap bson.M, asColumnName, function, column string) bson.M {
+	if column != "*" {
```

Comment:
When does this case occur ?
Also make sure you fix #1063

Reply:
This case occurs when the aggregate function is not 'count'

Reply:
when do you set **column to *** ?

Reply:
I think this pr doesn't fix #1063